src/heuristic/HeuristicMapper.cpp

```diff
@@ -597,3 +615,162 @@ void HeuristicMapper::lookahead(const std::size_t      layer,
                                          // for better fidelity lookahead
   }
 }
+
+void HeuristicMapper::Node::applySWAP(const Edge& swap, Architecture& arch) {
+  nswaps++;
```

Comment:
```suggestion
  ++nswaps;
```
small nit. probably doesn't even make a difference because the compiler rewrites it anyway.